services/web/server/src/simcore_service_webserver/resource_usage/_pricing_plans_handlers.py

```diff
@@ -69,3 +74,77 @@ async def get_pricing_plan_unit(request: web.Request):
     )
 
     return envelope_json_response(webserver_pricing_unit_get, web.HTTPOk)
+
+
+@routes.get(
+    f"/{VTAG}/pricing-plans",
+    name="list_pricing_plans",
+)
+@login_required
+@permission_required("resource-usage.read")
+@_handle_resource_usage_exceptions
+async def list_pricing_plans(request: web.Request):
+    req_ctx = RequestContext.model_validate(request)
+
+    pricing_plans_list = await admin_api.list_pricing_plans(
+        app=request.app,
+        product_name=req_ctx.product_name,
+    )
+    webserver_pricing_unit_get = [
+        PricingPlanGet(
```

Comment:
As agreed, please create the adapter as a static member `PricingPlanGet.from_domain_model`